mbta.rb

```diff
@@ -0,0 +1,52 @@
+def method(start_line, finish_line, start_stop, finish_stop)
+mbta = {
+  red_line: {
+    alewife: 6,
+    davis: 5,
+    porter: 4,
+    harvard: 3,
+    central: 2,
+    kendall: 1,
+    park_street: 0,
+    south_station: -1
+  },
+  green_line: {
+    haymarket: 2,
+    government_center: 1,
+    park_street: 0,
+    boylston: -1,
+    arlington: -2,
+    copley: -3
+  },
+  orange_line: {
+    north_station: 2,
+    haymarket: 1,
+    park_street: 0,
+    state_street: -1,
+    downtown_crossing: -2,
+    chinatown: -3,
+    tufts_medical_center: -4
+  }
+}
```

Comment:
Minor point: Having a hash of arrays might have been easier, since arrays have index numbers. Nothing wrong with this choice, though.